src/EnergyPlus/CurveManager.cc

```diff
@@ -2248,13 +2249,22 @@ namespace Curve {
                             lookupValues.push_back(Numbers(1));
                         }
 
-                        std::vector<Btwxt::GridAxis> gridAxes;
-                        gridAxes.emplace_back(axis, Btwxt::Method::LINEAR, Btwxt::Method::LINEAR, std::pair<double, double>{0.0, 360.0});
-
-                        auto gridIndex = // (AUTO_OK_OBJ)
-                            state.dataCurveManager->btwxtManager.addGrid(Alphas(1), Btwxt::GriddedData(gridAxes));
-                        thisCurve->TableIndex = gridIndex;
-                        thisCurve->GridValueIndex = state.dataCurveManager->btwxtManager.addOutputValues(gridIndex, lookupValues);
+                        try {
+                            std::vector<Btwxt::GridAxis> gridAxes;
+                            gridAxes.emplace_back(axis,
+                                                  "",
+                                                  Btwxt::Method::linear,
+                                                  Btwxt::Method::linear,
+                                                  std::pair<double, double>{0.0, 360.0},
+                                                  BtwxtManager::btwxt_logger);
+
+                            auto gridIndex = // (AUTO_OK_OBJ)
+                                state.dataCurveManager->btwxtManager.addGrid(Alphas(1), gridAxes);
+                            thisCurve->TableIndex = gridIndex;
+                            thisCurve->GridValueIndex = state.dataCurveManager->btwxtManager.addOutputValues(gridIndex, lookupValues);
+                        } catch (Btwxt::BtwxtException &e) {
+                            ShowSevereError(state, "GridAxis construction error.");
```

Comment:
Should this lead to a fatal error?  Does it?

Reply:
Yes, and fixed.